src/components/DisplayComponent.js

```diff
@@ -14,6 +24,33 @@ class DisplayComponent extends Component {
   }
 
   render() {
+    var display;
+    if (this.state.displayComponent === 'Chart') {
+      display = (
+        <BarChart
+          width={600}
+          height={300}
+          data={[
+            { name: 'Page A', uv: 4000, pv: 2400, amt: 2400 },
```

Comment:
I'm assuming these are just placeholder data, and will be removed as we move forward.

Reply:
yes! this is just placeholder data. I will remove it once I tie it up with the api results